CHANGELOG.md

```diff
@@ -110,6 +110,8 @@ Please also have a look at our
 
 ### Fixed
 
+- Selector functions (like `:not`) with comma-separated arguments are now
+  parsed correclty (#1292)
```

Comment:
```suggestion
  parsed correctly (#1292)
```

Reply:
I must be somewhat dyslexic; I never seem to spot these mistakes myself :)